src/managed/OpenLiveWriter.BlogClient/Clients/MediumClient.cs

```diff
@@ -0,0 +1,199 @@
+		private IBlogClientOptions _clientOptions;
+
+		public bool IsSecure
+		{
+			get
+			{
+				try
+				{
+					return String.Equals(_postApiUri.Scheme, "https", StringComparison.InvariantCultureIgnoreCase);
+				}
+				catch
+				{
+					return true;
+				}
+			}
+		}
+
+		public IBlogClientOptions Options
+		{
+			get
+			{
+				return _clientOptions;
+			}
+		}
+
+		public MediumClient(Uri postApiUrl, IBlogCredentialsAccessor credentials) : base(credentials)
+		{
+			_postApiUri = new Uri(UrlHelper.SafeToAbsoluteUri(postApiUrl));
+			var clientOptions = new BlogClientOptions
+			{
+				SupportsCategories = true,
+				SupportsMultipleCategories = true
+			};
+
+			_clientOptions = clientOptions;
+		}
+
+		public string AddCategory(string blogId, BlogPostCategory category)
+		{
+			throw new NotImplementedException();
+		}
+
+		public void DeletePage(string blogId, string pageId)
+		{
+			throw new NotImplementedException();
+		}
+
+		public void DeletePost(string blogId, string postId, bool publish)
+		{
+			throw new NotImplementedException();
+		}
+
+		public void DoAfterPublishUploadWork(IFileUploadContext uploadContext)
+		{
+			throw new NotImplementedException();
+		}
+
+		public string DoBeforePublishUploadWork(IFileUploadContext uploadContext)
+		{
+			throw new NotImplementedException();
+		}
+
+		public bool? DoesFileNeedUpload(IFileUploadContext uploadContext)
+		{
+			throw new NotImplementedException();
+		}
+
+		public bool EditPage(string blogId, BlogPost page, bool publish, out string etag, out XmlDocument remotePost)
+		{
+			throw new NotImplementedException();
+		}
+
+		public bool EditPost(string blogId, BlogPost post, INewCategoryContext newCategoryContext, bool publish, out string etag, out XmlDocument remotePost)
+		{
+			throw new NotImplementedException();
+		}
+
+		public AuthorInfo[] GetAuthors(string blogId)
+		{
+
+
+			return null;
+		}
+
+		public BlogPostCategory[] GetCategories(string blogId)
+		{
+			throw new NotImplementedException();
+		}
+
+		public BlogInfo[] GetImageEndpoints()
+		{
+			throw new NotImplementedException();
+		}
+
+		public BlogPostKeyword[] GetKeywords(string blogId)
+		{
+			throw new NotImplementedException();
+		}
+
+		public BlogPost GetPage(string blogId, string pageId)
+		{
+			throw new NotImplementedException();
+		}
+
+		public PageInfo[] GetPageList(string blogId)
+		{
+			throw new NotImplementedException();
+		}
+
+		public BlogPost[] GetPages(string blogId, int maxPages)
+		{
+			throw new NotImplementedException();
+		}
+
+		public BlogPost GetPost(string blogId, string postId)
+		{
+			throw new NotImplementedException();
+		}
+
+		public BlogPost[] GetRecentPosts(string blogId, int maxPosts, bool includeCategories, DateTime? now)
+		{
+			throw new NotImplementedException();
+		}
+
+		public BlogInfo[] GetUsersBlogs()
+		{
+			var httpWebRequest = (HttpWebRequest)WebRequest.Create(String.Concat(MediumApiUri, "/users/" + "/publications"));
```

Comment:
we can use String.Concat(object, object, object) instead of mixing concat and +.